### PR TITLE
LetsDoeIt - Remove extra text in title

### PR DESCRIPTION
**Pull request by Muldec**
An extra test is appended after the title of every video. This removes it.

### CODE REVIEW

File: scrapers/LetsDoeIt.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ xPathScrapers:
         selector: //title/text()
         postProcess:
           - replace:
-              - regex: (.+)(?:\s\|.+)
+              - regex: (.+)(?:Exclusive Porn Video)(?:\s\|.+)
                 with: $1
```
Comment (bnkai) [inline scrapers/LetsDoeIt.yml:24]: In cases where the `Exclusive Porn Video` is not present or is a bit different eg dirtycosplay the above breaks the match
I think adding a second regex removing the extra part is safer
```yaml
        postProcess:
          - replace:
              - regex: (.+)(?:\s\|.+)
                with: $1
              - regex: (Exclusive\s(\S+\s)?Porn\sVideo)
                with: ""
```

Comment (Muldec) [reply scrapers/LetsDoeIt.yml:24]: You're right !

Wouldn't `(Exclusive\W+Porn\W+Video)` be better ?

Comment (bnkai) [reply scrapers/LetsDoeIt.yml:24]: If you mean `(Exclusive\W+(\w+\W+)?Porn\W+Video)` that also works
without the extra match group like you have it, it fails in the dirtycosplay case

Comment (Muldec) [reply scrapers/LetsDoeIt.yml]: Right !
File updated and Squashed.

**Pull request merged**